packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.tsx

```diff
@@ -93,14 +90,30 @@ export type QueryWrapperPlotNavProps = SearchParams &
   Operator &
   QueryWrapperPlotNavOwnProps
 
-const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> = (
-  props: QueryWrapperPlotNavProps,
-) => {
-  const [showExportMetadata, setShowExportMetadata] = React.useState(false)
+type QueryWrapperPlotNavContentsProps = Pick<
+  QueryWrapperPlotNavProps,
+  | 'tableConfiguration'
+  | 'name'
+  | 'cardConfiguration'
+  | 'facetsToPlot'
+  | 'availableFacets'
+  | 'hideDownload'
+  | 'hideQueryCount'
+  | 'hideSqlEditorControl'
+  | 'searchConfiguration'
+  | 'showExportToCavatica'
+  | 'cavaticaHelpURL'
+  | 'customControls'
+  | 'fileIdColumnName'
+  | 'fileNameColumnName'
+  | 'fileVersionColumnName'
+> & {
+  isFullTextSearchEnabled: boolean
+  remount: () => void
+}
+
+function QueryWrapperPlotNavContents(props: QueryWrapperPlotNavContentsProps) {
```

Comment:
Contents of `QueryWrapperPlotNav` were extracted into an individual component because we won't be able to use the `Context.Consumer` pattern with the new state management tool.